src/playlist.js

```diff
@@ -374,9 +374,9 @@ export const getMediaInfoForTime = function(
       for (let i = startIndex - 1; i >= 0; i--) {
         const segment = partsAndSegments[i];
 
-        time += (segment.duration + TIME_FUDGE_FACTOR);
+        time += segment.duration;
 
-        if (time > 0) {
+        if ((time + TIME_FUDGE_FACTOR) > 0) {
```

Comment:
Should we keep `TIME_FUDGE_FACTOR` at all?

Reply:
question: What is `TIME_FUDGE_FACTOR` used for here? Is using it causing timing issues down the line?

Reply:
I think this change basically encompasses what you are saying. We only use `getMediaIndexForTime` when we are not incrementing mediaIndex/partIndex. This change makes it so that cumulatively we can only be off by `TIME_FUDGE_FACTOR` rather than every single part/segments duration being incremented or decremented by `TIME_FUDGE_FACTOR`. For the sake of time I think that we might want to add a TODO here to see if we should even use TIME_FUDGE_FACTOR at all. I lean towards not using it, but it will require a bit more testing and could easily enough be an isolated change.

Reply:
👍 a TODO would be good for now. This definitely is better than before.

Reply:
Probably worth making an issue/a note in our major/refactoring epic

Reply:
added